exporter/coralogixexporter/logs_client.go

```diff
@@ -56,8 +56,16 @@ type logsExporter struct {
 }
 
 func (e *logsExporter) start(ctx context.Context, host component.Host) (err error) {
```

Comment:
Why not invert the logic and avoid the negative at the beginning of the statement, it may make readability a little better.

Like:

```go
switch {
	case isEmpty(e.config.Logs.Endpoint):
		if e.clientConn, err = e.config.getDomainGrpcSettings().ToClientConn(ctx, host, e.settings, grpc.WithUserAgent(e.userAgent)); err != nil {
			return err
		}
	case isEmpty(e.config.Domain):
		if e.clientConn, err = e.config.Logs.ToClientConn(ctx, host, e.settings, grpc.WithUserAgent(e.userAgent)); err != nil {
			return err
		}
}
```

Since we're validating it before we create a connection should be fine